mgmt/rest/v2/api.go

```diff
@@ -0,0 +1,74 @@
+package v2
+
+import (
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/intelsdi-x/snap/pkg/api"
+)
+
+const (
+	version = "v2"
```

Comment:
It's a global constant, which is more reasonable than a global variable because it can't change. (the compiler will inline the constant everywhere it's used)